index.html

```diff
@@ -18,7 +18,11 @@
   <script>
     (function(window, videojs) {
       var player = window.player = videojs('videojs-vr-player');
-      player.vr({projection: '360', debug: true});
+      player.mediainfo = player.mediainfo || {};
```

Comment:
I don't love the way `mediainfo` has leaked out of the Brightcove Player and into a couple of our open source projects. Is there a different/better way?